### ISSUE TITLE
Choosing a music directory with no audio files causes crash and requires music_prefs.cfg to be deleted to re-open the player

### ISSUE THREAD

**Issue by fegkay15** [FishPhd/FishPlayer#8]
I chose a directory that had no audio files of it's own, but had 7 other directories in it which contained the audio files I was wanting to play. Quick fix would maybe be to catch the error of no audio files and to say that here are no audio files in the directory chosen. Suggested fix would be to maybe check every directory contained within the chosen directory for audio files as well and if there are directories in those, check those as well until all child directories are checked. Hopefully that wasn't too lengthy of an explanation.

**Comment by FishPhd** [comment]
Do you have a picture of your path? I am having a hard time replicating this.

**Comment by fegkay15** [comment]
I chose "..Downloads\Halo Audiobooks" as my music directory and the player crashed.
![image](https://cloud.githubusercontent.com/assets/13324054/10874507/c6708592-80f3-11e5-9f25-702362b08d0e.png)
![image](https://cloud.githubusercontent.com/assets/13324054/10874511/d5ea4b48-80f3-11e5-835d-dbadfacc928f.png)

**Comment by fegkay15** [comment]
But if I choose lets say "\Halo - Contact Harvest" as my music directory, the player does not crash and I can play the audio files just fine.

**Comment by fegkay15** [comment]
I apparently didn't troubleshoot well enough. I went and put an mp3 file in the main directory and it didn't work. It then dawned on me that maybe my uTorrent client was still using either the main directory or one of the sub-directories. So I closed uTorrent out and tried it again and everything worked like normal. I opened uTorrent back up to see if it would crash the player again. But it didn't. So the problem is no longer occuring, but I can't replicate it anymore. Sorry for misdiagnosing an issue.
